documentation/Doxyfile-mcss

```diff
@@ -0,0 +1,5 @@
+@INCLUDE                = Doxyfile
+GENERATE_HTML           = NO
+GENERATE_XML            = YES
+XML_PROGRAMLISTING      = NO
+OUTPUT_DIRECTORY        = ../build/doc/doxygen
```

Comment:
We can probably just merge this into the main Doxyfile as the intended use is with mcss only anyways.